site/docs/how-to-vote.md

```diff
@@ -0,0 +1,74 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Voting on an Iceberg Release
+
+Before a new version of Apache Iceberg is released, a release candidate will be prepared and a vote
+will occur. Announcements will be made on the Iceberg [developer's mailing list](http://127.0.0.1:8000/#community/#mailing-lists)
```

Comment:
Does this need localhost with the new docs setup or can we use relative links?

Reply:
Oops! Good catch, that was a copy-pasta mistake, lol

Reply:
Fixed!